src/styles/card.scss

```diff
@@ -0,0 +1,101 @@
+.card {
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  font-size: 12px;
+  padding: 32px 16px 16px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
```

Comment:
Delete align-items property from the card and you can delete width: 100% for all block elements, try it